### ISSUE TITLE
Invalid attribute 'quiet'?

### ISSUE THREAD

**Issue by triplingual** [inpho/topic-explorer#166]
I'm not quite sure where to go with this.
`topicexplorer init path/to/files config-name.ini` worked fine (though I initially specified the config-name file as .cfg rather than .ini and had to rename), but then `topicexplorer train config-name.ini` resulted in the below. Install of topic-explorer was today, if that matters, cloned off the head.

```
Traceback (most recent call last):
  File "//anaconda/bin/topicexplorer", line 9, in <module>
    load_entry_point('topicexplorer==1.0b132', 'console_scripts', 'topicexplorer')()
  File "//anaconda/lib/python2.7/site-packages/topicexplorer/__main__.py", line 218, in main
    benchmark(train.main)(args)
  File "//anaconda/lib/python2.7/site-packages/topicexplorer/train.py", line 171, in main
    if args.iter is None and not args.quiet:
AttributeError: 'Namespace' object has no attribute 'quiet'

```

**Comment by duhaime** [comment]
We hit the same trouble earlier today, and just changed line 171 to:

```
if args.iter is None:
```

That is, skip the part that looks for a quiet attribute on the args object, and you may be able to proceed. This is a hack that allowed one team member to carry on earlier today...

**Comment by JaimieMurdock** [comment]
Thanks for the bug report! I just pushed out a release to PyPI - running `topicexplorer update` should fix the issue. This was introduced by incomplete testing of a fix to #99.

Let me know if there are any other problems.
